README.md

```diff
@@ -4,7 +4,6 @@
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![CocoaPods](https://img.shields.io/cocoapods/v/MapboxDirections.svg)](https://cocoapods.org/pods/MapboxDirections/)
 [![SPM compatible](https://img.shields.io/badge/SPM-compatible-4BC51D.svg?style=flat)](https://swift.org/package-manager/)
-[![codecov](https://codecov.io/gh/mapbox/mapbox-directions-swift/branch/main/graph/badge.svg?token=bjr4ASuXuO)](https://codecov.io/gh/mapbox/mapbox-directions-swift)
```

Comment:
Don't we want to mention `lcov`?

Reply:
This section displayed labels on the top of the repo, to show the coverage. Unfortunately, `lcov` is just a local tool to build the report, not to provide the actual coverage status and coverage history.

<img width="690" alt="Screenshot 2025-02-06 at 11 26 26" src="https://github.com/user-attachments/assets/58dafd44-1fa0-4231-9603-dc896bd91a00" />